scripts/release-tools/downstream/y-stream-setup.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+set -eoux pipefail
+
+previous_release=$1
+release=$2
+
+get_supported_versions() {
+    supported_json="$(curl https://access.redhat.com/product-life-cycles/api/v1/products?name=Red%20Hat%20Advanced%20Cluster%20Security%20for%20Kubernetes)"
+    
+    supported_versions=($(echo $supported_json | jq -r '.data[0].versions[] | select(.type == "Full Support") | .name'))
+    
+    nversions=${#supported_versions[@]}
+    for ((i = nversions - 1; i >= 0; i = i - 1)); do
+        versions+=(${supported_versions[$i]})
+    done
+    
+    versions+=("$release")
+    nversions=${#versions[@]}
+}
```

Comment:
What does this function return? I guess this is `versions` variable?
I believe the function should operate on local vars only and do a proper return - here, we should echo the array, so that the caller may consume it.

Reply:
Previously it didn't return anything. Now it returns an array of versions.